contracts/interfaces/ISwap.sol

```diff
@@ -50,6 +50,22 @@ interface ISwap {
         bytes32 s;
     }
 
+    struct OfferDetails {
+        // 32 byte slot 1, partial fill
+        // Seller wallet address
+        address seller;
+        // 32 byte slot 2
+        // Addess of oToken
+        address oToken;
+        // Price per oToken denominated in biddingToken
+        uint96 minPrice;
```

Comment:
It's better to use `uint256` because EVM operates in words (32 bytes), so it saves gas